### PR TITLE
Add a clip board to every lawyer office

### PR DESCRIPTION
**Pull request by Certified-Air-Breather**
## About The Pull Request
Adds a clip board to all of the lawyer offices, usually on a table or rack,

## Why It's Good For The Game
Lawyers should have easy access to clipboards as I find it to be a better way to manage papers than folders

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>
Im not sure what exactly I am supposed to put here, first time contributing
Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog
:cl:
add: Added new things
add: Added more things
del: Removed old things
tweak: tweaked a few things
balance: rebalanced something
fix: fixed a few things
soundadd: added a new sound thingy
sounddel: removed an old sound thingy
imageadd: added some icons and images
imagedel: deleted some icons and images
spellcheck: fixed a few typos
code: changed some code
refactor: refactored some code
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know
/:cl:

### CODE REVIEW

**Comment by Certified-Air-Breather** [comment]
Im realizing why its called development hell, I got no clue what im doing.

File: code/game/objects/items/clown_items.dm [hunk 1]
```diff
@@ -126,6 +126,7 @@
 				REMOVE_TRAIT(target, TRAIT_SPRAYPAINTED, CRAYON_TRAIT)
 			target.wash(CLEAN_SCRUB)
 			target.remove_atom_colour(WASHABLE_COLOUR_PRIORITY)
+			addfingerprint("Traces of Soap") // Yay my first PR ! This makes it so soap leaves a trace
 			decreaseUses(user)
```
Comment (ToasterBan) [inline code/game/objects/items/clown_items.dm:130]: seems like you accidentally added this to your PR?

**Comment by ToasterBan** [comment]
all the tags are because you didn't properly do your body of the pull request
the changelog should look like this:
```
 ## Changelog
 🆑 
 add: Clipboards to every lawyer office
 /:cl:
```

and you should put in some screenshots of the lawyer offices with clipboards in the screenshots section, you also seem to have put the changelog inside the  `<details>` `</details>`  of the screenshots section

File: code/game/objects/items/clown_items.dm (hunk 1 above)
Comment (Certified-Air-Breather) [reply code/game/objects/items/clown_items.dm:130]: I did, I dont know why it still thought I was on traces of soap

**Comment by Certified-Air-Breather** [comment]
Nevermind I think I should just take a break and learn coding some more

**Comment by BlueHNT** [comment]
> Nevermind I think I should just take a break and learn coding some more

Don't worry about it too much. Even I had to start somewhere... I'm still nowhere.

**Pull request closed without merging**